### PR TITLE
Features/ncne data structure

### PR DESCRIPTION
**Pull request by ssrajan**
Creating the following
1. Local database with new task table
2. EF Project
3. Test project
4. Build and Release pipeline for database.

### CODE REVIEW

File: src/Databases/NCNEWorkflowDatabase.EF/Models/NcneTaskInfo.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace NCNEWorkflowDatabase.EF.Models
+{
+    public class NcneTaskInfo
+    {
+        [Key]
+        [DatabaseGenerated(databaseGeneratedOption: DatabaseGeneratedOption.Identity)]
```
Comment (Rossall82) [inline src/Databases/NCNEWorkflowDatabase.EF/Models/NcneTaskInfo.cs]: Was there a specific reason why this was needed?

Comment (ssrajan) [reply src/Databases/NCNEWorkflowDatabase.EF/Models/NcneTaskInfo.cs]: It was needed to treat the field name as primary key if the name of the field is other than Id.

File: src/NCNEPortal/Startup.cs [hunk 2]
```diff
@@ -74,8 +76,21 @@ public void ConfigureServices(IServiceCollection services)
                 options.TokenValidationParameters.ValidateIssuer = false; // accept several tenants (here simplified for development - TODO)
             });
 
+            var workflowDbConnectionString = DatabasesHelpers.BuildSqlConnectionString(isLocalDevelopment,
+                    isLocalDevelopment ? startupConfig.LocalDbServer : startupConfig.WorkflowDbServer, "NCNEWorkflowDatabase");
```
Comment (UKHOGreg) [inline src/NCNEPortal/Startup.cs]: The hardcoded string `"NCNEWorkflowDatabase"` should ideally be replaced with something like `startupConfig.NCNEWorkflowDbName`

Comment (ssrajan) [reply src/NCNEPortal/Startup.cs]: Added into the config

**Pull request merged**